oat-sa/extension-tao-itemqti
# Added ItemContentCloned event listener that rewrite item id in qti.xml

Pull request summary
 
Related to : https://oat-sa.atlassian.net/browse/UNO-547
 
There is a new listener that will update items identifier to just created in the qti.xml file when ItemContentClonedEvent triggers.
 
#### How to test
 
- Click the copy/clone button in the backoffice, items tab.
- Authorize the item and on the right panel check identifier input

without fix it will be the same as in the resource
with fix it will be the new identifier of the just created item
 
Test commands :
 
```
sudo -u www-data ./vendor/bin/phpunit taoQtiItem\test\unit\model\listener\QtiXmlDataManagerTest
```
 
#### Dependencies
 
Requires :
 - [x] https://github.com/oat-sa/extension-tao-item/pull/425
